ovos_utils/ocp.py

```diff
@@ -0,0 +1,518 @@
+    # The current media has been loaded. PlayerState== STOPPED
+    LOADED_MEDIA = 3
+    # Playback of the current media has stalled due to
+    # insufficient buffering or some other temporary interruption.
+    # PlayerState != STOPPED
+    STALLED_MEDIA = 4
+    # The player is buffering data but has enough data buffered
+    # for playback to continue for the immediate future.
+    # PlayerState != STOPPED
+    BUFFERING_MEDIA = 5
+    # The player has fully buffered the current media. PlayerState != STOPPED
+    BUFFERED_MEDIA = 6
+    # Playback has reached the end of the current media. PlayerState == STOPPED
+    END_OF_MEDIA = 7
+    # The current media cannot be played. PlayerState == STOPPED
+    INVALID_MEDIA = 8
+
+
+class PlayerState(IntEnum):
+    # https://doc.qt.io/qt-5/qmediaplayer.html#State-enum
+    STOPPED = 0
+    PLAYING = 1
+    PAUSED = 2
+
+
+class LoopState(IntEnum):
+    NONE = 0
+    REPEAT = 1
+    REPEAT_TRACK = 2
+
+
+class PlaybackType(IntEnum):
+    SKILL = 0  # skills handle playback whatever way they see fit,
+    # eg spotify / mycroft common play
+    VIDEO = 1  # Video results
+    AUDIO = 2  # Results should be played audio only
+    AUDIO_SERVICE = 3  ## DEPRECATED - used in ovos 0.0.7
+    MPRIS = 4  # External MPRIS compliant player
+    WEBVIEW = 5  # webview, render a url instead of media player
+    UNDEFINED = 100  # data not available, hopefully status will be updated soon..
+
+
+class PlaybackMode(IntEnum):
+    AUTO = 0  # play each entry as considered appropriate,
+    # ie, make it happen the best way possible
+    AUDIO_ONLY = 10  # only consider audio entries
+    VIDEO_ONLY = 20  # only consider video entries
+    FORCE_AUDIO = 30  # cast video to audio unconditionally
+    FORCE_AUDIOSERVICE = 40  ## DEPRECATED - used in ovos 0.0.7
+    EVENTS_ONLY = 50  # only emit ocp events, do not display or play anything.
+    # allows integration with external interfaces
+
+
+class MediaType(IntEnum):
+    GENERIC = 0  # nothing else matches
+    AUDIO = 1  # things like ambient noises
+    MUSIC = 2
+    VIDEO = 3  # eg, youtube videos
+    AUDIOBOOK = 4
+    GAME = 5  # because it shares the verb "play", mostly for disambguation
+    PODCAST = 6
+    RADIO = 7  # live radio
+    NEWS = 8  # news reports
+    TV = 9  # live tv stream
+    MOVIE = 10
+    TRAILER = 11
+    AUDIO_DESCRIPTION = 12  # narrated movie for the blind
+    VISUAL_STORY = 13  # things like animated comic books
+    BEHIND_THE_SCENES = 14
+    DOCUMENTARY = 15
+    RADIO_THEATRE = 16
+    SHORT_FILM = 17  # typically movies under 45 min
+    SILENT_MOVIE = 18
+    VIDEO_EPISODES = 19  # tv series etc
+    BLACK_WHITE_MOVIE = 20
+    CARTOON = 21
+    ANIME = 22
+    ASMR = 23
+
+    ADULT = 69  # for content filtering
+    HENTAI = 70  # for content filtering
+    ADULT_AUDIO = 71  # for content filtering
+
+
+@deprecated("import ovos_utils.available_extractors from ovos_plugin_manager.ocp instead", "0.1.0")
```

Comment:
Already deprecated? Should this be removed before stable release?

Reply:
yeah, these were short lived only during a couple alpha versions of 0.1.0, just left it to be sure i didnt miss any references in ovos-media (only repo using it) so it's safe to delete now